include/mem_mng.h

```diff
@@ -0,0 +1,35 @@
+#ifndef MEM_MNG_H
+#define MEM_MNG_H
+
+#include <assert.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <sys/mman.h>
+
+#include <cheriintrin.h>
+
+#include "compartment.h"
+
+#define COMP_MEM_DDC_OFFSET -16
```

Comment:
Why these sizes? It might be better to define them in terms of `sizeof` (assuming there's a sensible type e.g. `__capability`?).

Reply:
Removed for now in https://github.com/capablevms/CHERI-ELF-comp/pull/1/commits/fffb818b9368862b475fd83226805894d054a774. They'll be readded shortly in another form.